nefertari/authentication/models.py

```diff
@@ -159,6 +159,9 @@ def random_uuid(**kwargs):
 def encrypt_password(**kwargs):
     """ Crypt :new_value: if it's not crypted yet. """
     new_value = kwargs['new_value']
+    if kwargs['event'].is_auth_view:
+        return new_value
+
```

Comment:
>  let's discuss this.

Ok

> password gets passed to login() then authenticate_by_password() which takes care of the encryption

authenticate_by_password() checks raw password against encrypted password; it doesn't decrypt passed password and I think it's not possible to decrypt the password - only compare raw with decrypted.

Also same value gives different encrypted string when using encryption.